src/components/calculator.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import TopSection from './topSection';
+import SideBar from './sideActions';
+import MainActions from './mainActions';
+
+const Holder = () => (
+  <section className="holder">
+    <TopSection initial={0} />
+    <MainActions />
+    <SideBar />
+  </section>
+);
+export default Holder;
```

Comment:
- [ ] [OPTIONA] Please, you should try as much as possible to ensure that the function name is in line with file name for proper code maintainance ☺️.

Reply:
@AlphaNtihinduka Thankyou for the kind reviewer i will have this fixed 👍